File: Business Logic Layer(Backend)/Reservation/Reservation.Core/Data/Testimonial.cs
using System;
using System.Collections.Generic;
namespace Reservation.Core.Data
{
    public partial class Testimonial
    {
        public decimal Testimonial_Id { get; set; }
        public string Testimonial_Message { get; set; }
        public decimal? Customer_Id { get; set; }

        public virtual Customer Customer { get; set; }
    }
}

File: Presentation Layer (frontend)/Reservation_angular/src/data/testimonial.ts
export class Testimonial {
    private  testimonial_Id: number=-1;
    private testimonial_Message: string='';
    private customer_Id: number | null=null;
  
    
public get Testimonial_Id() : number {return this.testimonial_Id; }
public set Testimonial_Id(v :  number) {this.testimonial_Id = v;}
public get Testimonial_Message() : string {return this.testimonial_Message; }
public set Testimonial_Message(v :  string) {this.testimonial_Message = v;}
public get Customer_Id() : number | null {return this.customer_Id; }
public set Customer_Id(v :  number | null) {this.customer_Id = v;}
}